web_widget_json_graph/static/src/js/web_widget_json_graph.js

```diff
@@ -1,41 +1,23 @@
+/* eslint no-undef: 0*/
 odoo.define("web.web_widget_json_graph", function (require) {
     "use strict";
 
+    var ajax = require("web.ajax");
     var AbstractField = require("web.AbstractField");
     var field_registry = require("web.field_registry");
 
     var JSONGraphWidget = AbstractField.extend({
-        template: "JSONGraph",
-        cssLibs: ["/web/static/lib/nvd3/nv.d3.css"],
-        jsLibs: [
-            "/web/static/lib/nvd3/d3.v3.js",
-            "/web/static/lib/nvd3/nv.d3.js",
-            "/web/static/src/js/libs/nvd3.js",
-        ],
-        _render: function () {
+        jsLibs: ["/web/static/lib/Chart/Chart.js"],
```

Comment:
Can you separate the list??

```javascript
jsLibs: [
    "/web/static/lib/Chart/Chart.js",
],
```


So if more libs are added later on just a line is added and they don't have to edit your line to add the comma, break the line, and add another line, also it helps keep a better track of commits

Reply:
This is black I think. If so, it can't be avoided

Reply:
Drop, it's JS code

Reply:
Ignored

Reply:
Note: This change cannot be applied because prettier returns it to the current format.

Reply:
Ohh, ok, It's ok 😄